.npmignore

```diff
@@ -0,0 +1,2 @@
+.github/
+tests
```

Comment:
You've added tests, but what about example, README, .travis and other tech stuff?

Reply:
will be in another PR